packages/edit-site/src/components/dataviews/list-view.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { flexRender } from '@tanstack/react-table';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { chevronDown, chevronUp } from '@wordpress/icons';
+import { Button } from '@wordpress/components';
+import { forwardRef } from '@wordpress/element';
+
+const sortIcons = { asc: chevronUp, desc: chevronDown };
+function Header( { header } ) {
+	if ( header.isPlaceholder ) {
+		return null;
+	}
+	const text = flexRender(
+		header.column.columnDef.header,
+		header.getContext()
+	);
+	if ( ! header.column.getCanSort() ) {
+		return text;
+	}
+	const sortDirection = header.column.getIsSorted();
+	return (
+		<Button
+			onClick={ header.column.getToggleSortingHandler() }
+			icon={ sortIcons[ sortDirection ] }
+			iconPosition="right"
+			text={ text }
+			style={ { padding: 0 } }
+		/>
+	);
+}
+
+function ListView( { dataView, className, isLoading = false }, ref ) {
```

Comment:
This could be a bit of a question for design across some of this work, but is it possible for us to come up with a more unique name for the list view of table data, so that we don't confuse it with the `ListView` component in the sidebar of the post and site editors? Maybe prefixing with `Table` would be enough — `TableListView`?

Reply:
Yeah, we could rename, but this component will not be exposed and will only be used internally by the high level `DataViews` component.

Reply:
this could be named "TableView" maybe to avoid confusion, we'll have things like "GridView", "KanbanView"... later